download_logs.py

```diff
@@ -20,41 +23,101 @@ def get_arguments():
                         help='The url at which the server provides the dbinfo API.')
     parser.add_argument('--download-api', type=str, default="https://review.px4.io/download",
                         help='The url at which the server provides the download API.')
+    parser.add_argument('--mav_type', type=str, default=None, nargs = '+',
+                        help='Which mav types to download. Multiple are possible. Example: Quadrotor') # 1 +
+    parser.add_argument('--flight_modes', type=str, default=None, nargs='+',
+                        help='Whether to download only data with special flight modes '
+                             'Example: Mission')
+    parser.add_argument('--error_labels', default=None, nargs='+', type=str,
+                        help='Whether to only download data with special error labels '
```

Comment:
Can you specify that if you provide multiple, the log must have oll of the labels?